bedrock/base/templates/includes/sticky-promo.html

```diff
@@ -0,0 +1,40 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% set promo_referrals = '?utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=sticky-promo' %}
+
+<aside class="mzp-c-sticky-promo mzp-t-product-firefox mzp-t-dark hide-from-legacy-ie">
```

Comment:
Seeing as this adds new strings in a global file, should we only display the sticky promo it if the locale has the main headlines translated? (using `if (ftl_has_messages(): `? Right now it seems like this would show up in english everywhere right away.